Arduino/MiP_ProMini_Pack_Library/src/mip.cpp

```diff
@@ -2111,6 +2329,10 @@ uint8_t MiP::parseHexDigit(uint8_t digit)
 void MiP::processOobResponseData(uint8_t commandByte)
 {
     size_t length = 0;
+    size_t bytesRead;
+
+    uint8_t highNibble;
```

Comment:
I think you can get rid of highNibble and lowNibble now.

Reply:
Gone pending amended commit.